examples/psyad/eg1/testkern_mod.f90

```diff
@@ -34,31 +34,37 @@
 ! Author: A. R. Porter, STFC Daresbury Lab
 ! Modified by: R. W. Ford, STFC Daresbury Lab
 
-! Example module containing a very simple, one-line kernel subroutine.
+! Example module containing a very simple kernel subroutine that
+! contains only assignments.
```

Comment:
Please update as it now also contains a loop.

Reply:
Oh yes, done.